src/visualization/types/Graph/view.tsx

```diff
@@ -80,18 +84,9 @@ const XYPlot: FC<Props> = ({properties, result, timeRange, annotations}) => {
   )
   const dispatch = useDispatch()
 
-  // it doesn't know that it is even *in* a dashboard, much less which dashboard it is in.
-  // so having annotations on per dashboard is not supported yet
-  // these next two values are set in the dashboard, and used whether or not this view is in a dashboard
-  // or in configuration/single cell popout mode
```

Comment:
the first three lines of comments are definitely obsolete/not relevant any more,

but the last three lines references 'inAnnotationWriteMode", which is still relevant

Reply:
restored the relevant comments; and cleaned up the commits a little (made them all semantic; squashed two of them)

Reply:
```
// it doesn't know that it is even *in* a dashboard, much less which dashboard it is in.
// so having annotations on per dashboard is not supported yet
```

isn't really true. every cell operation happens in the context of a dashboard, so it's not that it does or doesn't know which dashboard it's in, it's just that it's irrelevant. it's not that per dashboard annotations aren't supported, it's that we chose not to do them in favor of cell-only annotations

Reply:
those two lines were not restored (because I agree that part is not relevant)